Mathlib/Data/Bool/Basic.lean

```diff
@@ -270,17 +269,22 @@ attribute [simp] not_or
 #align bool.bnot_inj Bool.not_inj
 
 instance linearOrder : LinearOrder Bool where
-  le_refl := Bool.le_refl
-  le_trans _ _ _ := Bool.le_trans
-  le_antisymm _ _ := Bool.le_antisymm
-  le_total := Bool.le_total
+  le_refl := by decide
+  le_trans := by decide
+  le_antisymm := by decide
+  le_total := by decide
   decidableLE := inferInstance
-  lt_iff_le_not_le _ _ := Bool.lt_iff_le_not_le
+  lt_iff_le_not_le := by decide
+  max_def := by decide
+  min_def := by decide
 #align bool.linear_order Bool.linearOrder
 
-attribute [simp] Bool.max_eq_or Bool.min_eq_and
+@[simp] theorem max_eq_or' : max = or := rfl
+
+@[simp] theorem min_eq_and' : min = and := rfl
```

Comment:
Should the Std lemmas be changed to match these ones instead?

Reply:
OK.

Reply:
(if the answer at the Std end is no, then thanks for adding these back; I don't think they should have been deleted in https://github.com/leanprover-community/mathlib4/pull/7982/files#r1375519160)

Reply:
I made them match and also simp in std4#329. The simp might be debatable in Std, I'm on the fence. Let's see what happens in Std and then fix here.